agent/credentials/instancecreds/instancecreds.go

```diff
@@ -38,7 +39,25 @@ func GetCredentials() *credentials.Credentials {
 	mu.Lock()
 	if credentialChain == nil {
 		credProviders := defaults.CredProviders(defaults.Config(), defaults.Handlers())
-		credProviders = append(credProviders, providers.NewRotatingSharedCredentialsProvider())
+		/*
```

Comment:
This comment repeats but also now contradicts the function signature comment above. Please remove the comment here and update the GetCredentials comment